metadata.rb

```diff
@@ -11,4 +11,4 @@
 supports 'centos'
 supports 'ubuntu'
 
-depends 'ark', '~> 5.0'
+depends 'ark', '5.0'
```

Comment:
This needs to be looser. 
```suggestion
depends 'ark', '>= 5.0'
```